### ISSUE TITLE
GGShield Binary Requires Too New of GLIBC

### ISSUE THREAD

**Issue by rgajason** [GitGuardian/ggshield#984]
## Environment

- ggshield version: 1.32.1
- Operating system (Linux, macOS, Windows): Linux
- Operating system version: RHEL 8, RHEL 9, UBI 8, UBI 9
- Python version: N/A

## Describe the bug

The GGShield binary is compiled to require GLIBC version 2.35 or higher.  That eliminates all RHEL, UBI, and RHEL clones as the most recent version of GLIBC available is 2.34.

```
$ ggshield --help
[708] Error loading Python lib '/usr/libexec/ggshield/_internal/libpython3.10.so.1.0': dlopen: /lib64/libm.so.6: version `GLIBC_2.35' not found (required by /usr/libexec/ggshield/_internal/libpython3.10.so.1.0)
```

As GLIBC is a core library it cannot be upgraded or an additional copy installed.

**Comment by agateau-gg** [comment]
Thank you for the report, we are looking into this.

**Comment by agateau-gg** [comment]
Fixed in 1.33.0.
